src/server/pps/pretty/pretty.go

```diff
@@ -164,7 +156,7 @@ Parallelism Spec: {{.ParallelismSpec}}
 	CPU: {{ .ResourceSpec.Cpu }}
 	Memory: {{ .ResourceSpec.Memory }} {{end}}
 Inputs:
-{{pipelineInputs .}}
```

Comment:
What about `inputs`?  If we are ok with people using pipelines that have `inputs` but not `input`, then I think we should support printing `inputs` as well.